geometry_msgs/Twist cmd
float64 durration
float64[] tau
float64[] q_start
float64[] q_end
float64[] qd_start
float64[] qd_end

